schemas/dex/schema.json

```diff
@@ -568,14 +568,6 @@
           "description": "The address that initiates the transaction (ie, the transaction signer).",
           "type": "string"
         },
-        "from_address": {
-          "description": "The from address of the event (ie, the from field in a transfer).",
-          "type": "string"
-        },
-        "to_address": {
-          "description": "The to address of the event (ie, the to field in a transfer).",
-          "type": "string"
-        },
```

Comment:
I get removing these. But could `transaction_from_address` not be the `user_address`?

Reply:
i kept the `transaction_from_address` for calculating DAU purposes .

the LP ownership will be identified by the transfer events. 

some details in this thread : https://openblockteam.slack.com/archives/C06GP5BEQ4C/p1726541797253409?thread_ts=1726534230.559209&cid=C06GP5BEQ4C